Dockerfile

```
@@ -0,0 +1,16 @@
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+
+ENV MODULE_NAME=license_manager_simulator.main
+
+RUN apt update && apt install -y curl
```

```suggestion
RUN apt-get update && apt-get install -y curl
```

`apt` should not be used in scripts, on some situations it requests actions from the user with non-scriptable actions.

---

I guess adding `ARG DEBIAN_FRONTEND=noninteractive` in the beginning of the file can overcome these situations, doesn't?